sdk/src/types/block/address/mod.rs

```diff
@@ -2,16 +2,17 @@
 // SPDX-License-Identifier: Apache-2.0
 
 mod alias;
+pub(crate) mod bech32;
```

Comment:
Doesn't need to be pub(crate) anymore since we don't need the serde symbol anymore ?